### PR TITLE
Add short-hand access to the rules

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Add a short-hand URL for accessing the rules, rather than srobo.org/docs/rules, or srobo.org/docs, then clicking rules.

If my understanding of the setup is correct, this should also allow srobo.org/rules to redirect to it.

### CODE REVIEW

File: _env/nginx.conf [hunk 1]
```diff
@@ -82,6 +82,10 @@ http {
       proxy_pass https://patience.studentrobotics.org/docs/python/;
     }
 
+    location /rules/ {
+      proxy_pass https://patience.studentrobotics.org/docs/rules/;
```
Comment (PeterJCLaw) [inline _env/nginx.conf]: I'm not sure we can spell this as we want using a `location` block. I think we want a `rewrite` tagged as a redirect (there's already a bunch of them lower down).

I think we want something like:
``` nginx
    rewrite ^/rules                /docs/rules permanent;
```

File: _env/nginx.conf [hunk 2]
```diff
@@ -81,7 +81,7 @@ http {
     location /docs/python/ {
       proxy_pass https://patience.studentrobotics.org/docs/python/;
     }
-
+
```
Comment (PeterJCLaw) [inline _env/nginx.conf]: unrelated change?

Comment (RealOrangeOne) [reply _env/nginx.conf]: Viiiiiiimmmmmm!
